src/Sulu/Bundle/MediaBundle/Media/PropertiesProvider/MediaPropertiesProvider.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\PropertiesProvider;
+
+use Symfony\Component\HttpFoundation\File\File;
+
+class MediaPropertiesProvider implements MediaPropertiesProviderInterface
+{
+    /**
+     * @var iterable<PropertiesProviderInterface>
+     */
+    private $providers;
+
+    public function __construct(iterable $providers)
+    {
+        $this->providers = $providers;
+    }
+
+    public function provide(File $file): array
```

Comment:
not sure if we need this service. we could just inject all providers into the `MediaManager` and do this there. that way, we would not need two interfaces

Reply:
@nnatter I don't like it that the user of the service need to handle the `array_merge`. I would really like a service which also can be used in imports which and return all properties.

Reply:
Orientated here by the `ContentMergerInterface` and `MergerInterface` from the ContentBundle. So `MediaPropertiesProviderInterface` and `PropertiesProviderInterface` 🙈 .

Reply:
do you think the user will need this service? 
using two interfaces is okay for me too, but then we should start to use this pattern everywhere 🙂 
for example, we did not use two interfaces for the `BlockVisitor` concept